assets/src/dashboard/utils/useFocusOut.js

```diff
@@ -17,9 +17,11 @@
 /**
  * External dependencies
  */
-import { useLayoutEffect } from 'react';
+import { useLayoutEffect, useRef } from 'react';
 
 function useFocusOut(ref, callback, deps) {
+  const mouseDownInsideTarget = useRef();
```

Comment:
Nit, if `mouseDownInsideTarget` is ref-ing a boolean, can you initialize the value so it's super clear the intent of this variable (when I first looked at the code I was scanning to see what dom element it was attached to):
```
const mouseDownInsideTarget = useRef(false);
```

Reply:
Also, to follow the naming of `isInDocument`, and `isInNode`, should this variable be called `isMouseDownInsideTarget` or even `isMouseDownInNode`?